app/src/components/nftCard.tsx

```diff
@@ -84,6 +84,8 @@ export const NftCard = (props: NftCardProps): React.ReactElement => {
                   <div dangerouslySetInnerHTML={{ __html: imageUrl.slice('data:image/svg+xml;utf8,'.length) }} />
                 ) : imageUrl.startsWith('data:image/svg+xml;base64,') ? (
                   <div dangerouslySetInnerHTML={{ __html: atob(imageUrl.slice('data:image/svg+xml;base64,'.length)) }} />
+                ) : imageUrl.includes('meebit') ? (
```

Comment:
can you include more of the url pls, like maybe the whole host so use imageUrl.startswith cos there could well be other projects that are meebit derivatives e.g. flippedmeebits